### ISSUE TITLE
Tab Panel note about arrow keys

### ISSUE THREAD

**Issue by aardrian** [w3c/aria-practices#976]
Arrow key support has been discussed repeatedly (#14, #40, #726). This is not a technical issue but an editorial one.

The [note](https://www.w3.org/TR/wai-aria-practices-1.1/#issue-container-generatedID-24) for the [tab panel](https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel) says:

> 2. If the tabs in a tab list are arranged vertically:
>   A. `Down Arrow` performs as `Right Arrow` is described above.
>   B. `Up Arrow` performs as `Left Arrow` is described above.

Below the note, under [WAI-ARIA Roles, States, and Properties](https://www.w3.org/TR/wai-aria-practices-1.1/#wai-aria-roles-states-and-properties-19) the final bullet reads:

> * If the `tablist` element is vertically oriented, it has the property `aria-orientation` set to `vertical`. The default value of `aria-orientation` for a `tablist` element is `horizontal`.

Suggest editing the note text to acknowledge the requirement for `aria-orientation`. For folks who do not read along (as I have seen), a vertical tab panel without `aria-orientation` may cause it to fail [1.3.3: Sensory Characteristics](https://www.w3.org/WAI/WCAG21/Understanding/sensory-characteristics.html). Suggested edit:

2. If the tabs in a tab list are arranged vertically **(both visually and by using `aria-orientation=vertical` on the tablist)**:

While I know it is incumbent on the developer to get this right, this text change can reduce that risk.

**Comment by mcking65** [comment]
This is a good suggestion. However

The aria-orientation property is supported in several patterns. If we are going to make this change, we should make it in all the patterns that use aria-orientation. Perhaps add a checklist of those patterns to this issue?

**Comment by aardrian** [comment]
aria-orientation is used in the roles:

* scrollbar
* select
* separator
* slider
* tablist
* toolbar

aria-orientation inherits into:

* combobox
* listbox
* menu
* menubar
* radiogroup
* tree
* treegrid

Of those roles, only the following exist in APG _and_ reference `aria-orientation`:

* [toolbar](https://www.w3.org/TR/wai-aria-practices-1.1/#toolbar), but does not use it in sample.
* [slider](https://www.w3.org/TR/wai-aria-practices-1.1/#slider), but arguably does not use it in the sample unless the lack of support for `PgUp` & `PgDn` on the horizontal sliders is intentional (feels like a bug).
* [slider (multi-thumb)](https://www.w3.org/TR/wai-aria-practices-1.1/#slidertwothumb), where it is not used in the sample but it does support `PgUp` & `PgDn` on the horizontal slider (which is part of why the lack of support on the regular slider feels like a bug).
* [listbox](https://www.w3.org/TR/wai-aria-practices-1.1/#Listbox), not used in the examples.
* [menu and menubar](https://www.w3.org/TR/wai-aria-practices-1.1/#menu), not used in the examples.
* [treeview](https://www.w3.org/TR/wai-aria-practices-1.1/#TreeView), not used in the examples.

In short, of the 6 existing APG patterns that mention `aria-orientation`, none of them use it (assuming the slider sample failure to support `PgUp` & `PgDn` is a bug).

So instead I suggest that when each of those patterns gets `aria-orientation` added to it, similar language come along.

**Comment by aardrian** [comment]
Of note, only menu and toobar use similar language to tab in their descriptor. I am happy to make PRs for those.
